src/inference/rejection.js

```diff
@@ -22,6 +21,17 @@ module.exports = function(env) {
       maxScore: 0,
       incremental: false
     });
+
+    this.throwOnError = options.throwOnError !== undefined ? options.throwOnError : true;
+    // the value of options.probe is the min sample rate
+    this.probe = options.probe;
```

Comment:
I think `options.probe: true` means both `minSampleRate: 250` and `throwOnError: false`. So I can replace `probe` with these two settings.